sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -1062,6 +1062,16 @@ class DDLParserSuite extends AnalysisTest {
       "It is not allowed to add catalog/namespace prefix a.b")
   }
 
+  test("UNCACHE TABLE table") {
```

Comment:
Duplicated `TABLE table`. "UNCACHE TABLE" or "UNCACHE table" would be correct.

Reply:
Ah OK. I see some discrepancies. I fixed other test names to have command as the test name.